static/bidder-info/zeta_global_ssp.yaml

```diff
@@ -1,5 +1,8 @@
-endpoint: https://ssp.disqus.com/bid/prebid-server?sid=GET_SID_FROM_ZETA
+disabled: true
+endpoint: https://ssp.disqus.com/bid/prebid-server?sid={{.AccountID}}
```

Comment:
endpoint is reachable;
![Uploading image.png…]()

Reply:
@przemkaczmarek I'm sorry, I didn't get it (cannot upload your img). Yes, the endpoint is reachable. And `sid` query param is not mandatory in current implementation. What is the issue here?

Reply:
@przemkaczmarek got it, thank you